src/bcrypt/__init__.py

```diff
@@ -103,3 +125,14 @@ def kdf(password, salt, desired_key_bytes, rounds):
 def _bcrypt_assert(ok):
     if not ok:
         raise SystemError("bcrypt assertion failed")
+
+
+def _compare(a, b):
```

Comment:
I'm not personally opposed to using a constant time comparison here if one is available, though I'd suggest maybe this should be written as:

```python
try:
    from hmac import compare_digest as _compare
except ImportError:
    def _compare(a, b):
        return a == b
```

This way we push the work of determining what comparison method is going to be used to a once-per-process import time cost instead of a once-per-comparison runtime cost.